docs/src/architecture/mapper.md

```diff
@@ -67,6 +67,48 @@ as both a fragment type and a fragment series in [Cumulocity's measurements](htt
 After the mapper publishes a message on the topic `c8y/measurement/measurements/create`,
 the message will be transferred to the topic `measurement/measurements/create` by [the MQTT bridge](../references/bridged-topics.md).
 
+### For child devices
+
+The Cumulocity mapper supports sending measurements to child devices under the `thin-edge.io` parent device.
+The incoming topic is different from the parent device.
+For child devices, use `tedge/measurements/<child-id>` topic. (`<child-id>` is your desired child device ID.)
+
+The mapper works in the following steps.
+
+1. When the mapper receives a Thin Edge JSON message on the `tedge/measurements/<child-id>`,
+   the mapper publishes the SmartREST `101` message to create a child device under the `thin-edge.io` parent device.
+   The name of the child device is the given child ID, and the type is `thin-edge.io-child`.
+2. Publish corresponded Cumulocity JSON measurements messages over MQTT.
+3. If the given child ID by the topic name is already known by the mapper,
```

Comment:
That's an implementation detail and I'm not sure if this level of detail is required in the userdoc. For the user, the relevant thing is that "the child device is created on receipt of the very first measurement for that child device".

Reply:
Agree with @albinsuresh . The user doc is not a specification. The proposed sentence is perfect.